src/web/mjs/connectors/ComicoNovel.mjs

```diff
@@ -16,6 +16,7 @@ export default class ComicoNovel extends Connector {
     }
 
     async _getPages(chapter) {
+        let darkmode =JSON.parse(Engine._settings.NovelDarkmode.value);
```

Comment:
Looks like redundant code when every consumer of the settings engine has to convert the value into the appropriate format.
I suggest to either make the [UI component](https://github.com/manga-download/hakuneko/blob/master/src/web/lib/hakuneko/frontend%40classic-dark/input.html#L57-L65) work with generic types instead of just strings, or use a string as identifier (object key) and provide a [global map](https://github.com/manga-download/hakuneko/blob/master/src/web/mjs/engine/Enums.mjs) that stores the color profiles for the given identifier.

```javascript
static get NovelColorProfiles() {
    return {
        // Sample Access: const colorProfile = Engine.Enums.NovelColorProfiles['dark']
        //                colorProfile.foreground
        dark: {
            foreground: '...',
            background: '...'
        },
        light: {
           // ...
        }
    };
}
```

Reply:
1st option, idk how to do
2nd option, seems like more work for both me(current dev), future dev(wants to add new colour) and, the computer(though im not sure about this)

Reply:
1st option doesn't seem possible